docs/api/IoT.js-API-WebSocket.md

```diff
@@ -0,0 +1,85 @@
+### Platform Support
+
+The following chart shows the availability of each WebSocket module API function on each platform.
+
+|  | Linux<br/>(Ubuntu) | Tizen<br/>(Raspberry Pi) | Raspbian<br/>(Raspberry Pi) | Nuttx<br/>(STM32F4-Discovery) | TizenRT<br/>(Artik053) |
+| :---: | :---: | :---: | :---: | :---: | :---: |
+| websocket.connect | O | X | X | X | X |
+| websocket.close | O | X | X | X | X |
+| websocket.ping | O | X | X | X | X |
+| websocket.send  | O | X | X | X | X |
+
+# WebSocket
+
+WebSocket provides full-duplex communication over a TCP connection. It is designed to work over HTTP ports 80 and 443.
+
+### Requirements
+WebSocket requires you to enable both the `TLS` and the `WebSocket` module. This can be done by compiling IoT.js with `--cmake-param=-DENABLE_MODULE_WEBSOCKET=1 --cmake-param=-DENABLE_MODULE_TLS=1`. Currently WebSocket only works if TLS is enabled as well.
```

Comment:
Why the TLS must be defined manually? 
Please use ON instead of 1: `--cmake-param=-DENABLE_MODULE_WEBSOCKET=ON`